### DISCUSSION TITLE
Status of BSP as an approved standard

### DISCUSSION THREAD

**Discussion by mnoah1** [build-server-protocol/build-server-protocol#658] [Q&A] [answered]
I am evaluating building a VS Code client for BSP, and wanted to ask for clarification on the [status](https://build-server-protocol.github.io/docs/specification#status) of BSP, which calls out that "The Build Server Protocol is not an approved standard."

As there are already various client and server implementations in progress, and there seem to be regular versioned releases, is this still the case?  What is your criteria for getting to the point at which BSP will be considered an approved standard (and by who)?

**Comment by jastice** [comment, marked as answer]
Good point. I don't think we need the blessing of any particular standards authority, so I'm fine with removing that wording.

**Comment by agluszak** [comment]
I don't think LSP is an "approved" standard either
